### DISCUSSION TITLE
Changing the title demo video to another programmatically

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#3812] [Q&A] [answered]
I like the way the game surprise me with the kickstarter BF toy commercial, it's pretty good, but i'd like to change it to a random funny vid i found on the internet by just changing the lines of the code, just for the sake of fun and experimentation, also for the future random/consecutive title demo vids feature, i don't want to wait only for the promo.

> peace
![cat-explosion-ezgif com-speed](https://github.com/user-attachments/assets/22433fc2-fd62-44fb-83fd-2c833bebd69b)

**Comment by KoloInDaCrib** [comment, marked as answer]
To do that you could go to` source/funkin/ui/title/AttractState.hx` and there you could change the video to be played upon inactivity !
Or if you're making a simple mod without any source code just put a .mp4 video titled "toyCommercial" in the `videos/videos` (yes that is two videos folders, one in the other) folder of your mod

**Comment by ghost** [comment]
One notice: the video should be in the 1280x720 resolution. Wishing you luck in modifying the code to your liking, i'll thank @KoloInDaCrib for the knowledge.
![df2qr03-fe423253-99f4-4099-886b-5981f1185458](https://github.com/user-attachments/assets/b29fdcc1-5bbd-4d5d-b676-bf425e3cd367)
